packages/orbit-components/src/HorizontalScroll/index.tsx

```diff
@@ -20,6 +19,35 @@ const getSnap = (scrollSnap: ScrollSnap) => {
   return scrollSnap;
 };
 
+const getTriggerOffset = (spacing: string) => {
```

Comment:
This completely changes the logic of the component, no? 🤔 the spacing was only for the space between elements. If you add it to the arrow positioning logic, it does not work...

Yours:
<img width="456" alt="image" src="https://github.com/user-attachments/assets/10231e7a-6203-4a8f-8480-2e9e502d190f">


Prod:
<img width="456" alt="image" src="https://github.com/user-attachments/assets/f9f444ac-9fd9-4e7c-9eb9-fbcf74e4d632">